src/SprykerEco/Yves/Payone/Plugin/PayoneBancontactOnlineTransferSubFormPlugin.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Payone\Plugin;
```

Comment:
Probably all other Eco modules have this mistake, but the plugin should repeat the path from the plugin interface. And in your case, it should be ...\Yves\Payone\Plugin\Form

Reply:
Whole module wrote in the same style. I think it's more relevant for new integration.